web-api/openapi/schemas/document.yml

```diff
@@ -28,8 +28,8 @@ DocumentPropertyArrayString:
   maxItems: 100
 
 DocumentProperty:
-  description: |
-    Arbitrary data that can be attached to a document. A subset of JSON is supported.
+  description: |-
+    Mostly arbitrary data that can be attached to a document. A subset of JSON is supported.
     When a property is set to null or an empty array, it is treated as if that property had no values.
     This means that they won't appear in searches with filters on these properties.
```

Comment:
❗ this description doesn't show up anywhere in the generated docs (likely because of the `oneOf`) and i didn't find a way to display it, also not via extensions. so we have to figure out if we can find a solution for that issue later on, since it contains important information.